client/src/containers/GameSelect.jsx

```diff
@@ -0,0 +1,14 @@
+import { connect } from 'react-redux';
+import GameSelect from '../pages/GameSelect';
+import { actionCreators } from '../actions';
+
+const mapStateToProps = state => ({
+  gameList: state.gameList,
+});
+
+const mapDispatchToProps = dispatch => ({
+  onCreateGame: () => dispatch(actionCreators.createGame()),
```

Comment:
`dispatch` is the parameter being passed into the `mapDispatchToProps` so when we pass this method to `connect` down below, redux makes this component subscribe to changes in the redux store and it will call this method again before with `dispatch` to get the props to pass to the GameSelect component